### PR TITLE
Gkc tweaks

### PR DESCRIPTION
**Pull request by notrandomness**

### CODE REVIEW

File: src/main/java/frc/robot/commands/AmpArmCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    @Override
+    public void execute(){
+        triggerAxis = controller.getRawAxis(trigger);
+        if (triggerAxis > Constants.stickDeadband){
+            ampArmSubsystem.isUp = true;
+            double speed = MathUtil.clamp(ampPIDController.calculate(ampArmSubsystem.getEncoderValue() - (ampArmSubsystem.lowLimit + 42)), -0.15, 0.5);
+            ampArmSubsystem.setSpeed(speed);
+            SmartDashboard.putNumber("PID Amp Output", speed);
```
Comment (notrandomness) [inline src/main/java/frc/robot/commands/AmpArmCommand.java]: We should comment this out to keep our dashboard clean

File: src/main/java/frc/robot/commands/ElevatorDefaultCommand.java [hunk 2]
```diff
@@ -53,48 +84,69 @@ public void execute(){
...
+        SmartDashboard.putNumber("ElevatorDegrees", angle);
 
         double degreesToEncoderAngle = (angle - Constants.Swerve.minElevatorAngle) * Constants.Swerve.degreesToEncoderValue;
 
-        //SmartDashboard.putNumber("Final Encoder Value", degreesToEncoderAngle);
+        SmartDashboard.putNumber("Final Encoder Value", degreesToEncoderAngle);
 
         double elevatorValue = elevatorSubsystem.getElevatorValue();  
+
+        SmartDashboard.putNumber("Current Encoder Value", elevatorValue);
```
Comment (notrandomness) [inline src/main/java/frc/robot/commands/ElevatorDefaultCommand.java:105]: We can probably comment out some of these dashboard outputs as well

File: src/main/java/frc/robot/subsystems/AmpArmSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    @Override
+    public void periodic(){
+        if (ampArmMotor.getForwardLimitSwitch(Type.kNormallyOpen).isPressed()){
+            highLimit = ampArmMotor.getEncoder().getPosition();
+            // ampArmMotor.getEncoder().setPosition(0);//TODO change this to the encoder value when amp arm is up
+        }
+        if (ampArmMotor.getReverseLimitSwitch(Type.kNormallyOpen).isPressed()){
+            lowLimit = ampArmMotor.getEncoder().getPosition();
+        }
+        SmartDashboard.putNumber("Amp Arm Encoder", getEncoderValue());
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/AmpArmSubsystem.java]: We can probably comment out this too

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 4]
```diff
@@ -203,8 +206,17 @@ public Pose2d getPose() {
     }
 
     public Pose2d getAutoLimelightBotPose(){
-        Pose2d visionPose = s_Limelight.getBotPose();
-        updateWithVisionLLEsitmator(visionPose, gyroOffset);
+        if (s_Limelight.getArea() >= 0.2) {
+            ChassisSpeeds speeds = Constants.Swerve.swerveKinematics.toChassisSpeeds(getModuleStates());
+            if (Math.abs(speeds.vxMetersPerSecond) < .25 && Math.abs(speeds.vyMetersPerSecond) < 1 && DriverStation.isAutonomous() && DriverStation.isEnabled()){
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Remove the isAutonomous and isEnabled parts of this

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 5]
```diff
@@ -203,8 +206,17 @@ public Pose2d getPose() {
...
+                Pose2d visionPose = s_Limelight.getBotPose();
+                updateWithVisionLLEsitmator(visionPose, s_Limelight.getLastBotPoseTimestamp());
+                System.out.println("Updating");
+            } else if (DriverStation.isEnabled() && DriverStation.isAutonomousEnabled()) {
+                System.out.println("NOT");
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Remove this else if and the System.outs

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 6]
```diff
@@ -415,7 +425,19 @@ public void periodic(){
...
+        SmartDashboard.putNumber("limelightBotPose Y", botPose.getY());
+        double driftDiffX = odometryX - botPose.getX();
+        double driftDiffY = odometryY - botPose.getY();
+        xDiffSum += driftDiffX;
+        yDiffSum += driftDiffY;
+        SmartDashboard.putNumber("Pos X Drift", driftDiffX);
+        SmartDashboard.putNumber("Pos Y Drift", driftDiffY);
+        SmartDashboard.putNumber("X Diff Sum", xDiffSum);
+        SmartDashboard.putNumber("Y Diff Sum", yDiffSum);
+        SmartDashboard.putNumber("Gyro Yaw", getGyroYaw().getDegrees());
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Comment out these dashboard outputs and the xDiff yDiff stuff.

**Comment by T-Wezzy** [comment]
gabapoool

**Review by T-Wezzy — DISMISSED** [review]
THIS CODE SUCKS!!!!!!!!!!! I HATE IT!!!!!! IT CAN BE OPTIMIZED BY LIKE 3 MILLISECONDS BUT YOU GUYS ARENT DOING IT GRRRRRRRRR

**Pull request merged**